lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(spot)
+    cells[spot.to_i - 1]
+  end
+
+  def full?
+    !(cells.include?(" "))
+  end
+
+  def turn_count
+    turns = cells.delete_if{|a| a == " "}
+    turns.count
```

Comment:
[`Array#count`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-count) accepts a block for unique counts and you could write:

```ruby
cells.size - cells.count{|c| c == " "}
```

or

```ruby
cells.count{|c| ["X", "O"].include?(" ")}
```